src/main/java/namor/template/main/App.java

```diff
@@ -1,9 +1,64 @@
 package namor.template.main;
 
+import java.util.Arrays;
+
 // TODO: 20.03.17 [GUSb] A test class to check everything works, should be deleted with first finished task
 public class App {
 
-    public int add(int a, int b) {
-        return a + b;
+    private int top;
+    int size;
+    int[] stack;
+
+    public App(int arraySize)
+    {
+        size = arraySize;
+        stack = new int[size];
+        top = 0;
     }
```

Comment:
Why only one constructor ? Why to force everybody to think about the size ? Make anothe one constructor with default size. So overload constructor to have two options in creation of this object (Polymorphysm)